history/characters/10000_orc.txt

```diff
@@ -291,7 +309,10 @@
 	# trait=education_martial_4 trait=sadistic
 	# trait=diligent trait=wrathful trait=ambitious 
 	# disallow_random_traits = yes
-	# 553.6.18={ birth=yes trait=creature_orc }
+	# 553.6.18={ 
+		birth=yes trait=creature_orc
+		# effect = { make_important_lore_character_effect = yes }
+	#}
```

Comment:
Check `error.log` before requesting reviews. One small typo can break everything.
```suggestion
	# 553.6.18={ 
		# birth=yes trait=creature_orc
		# effect = { make_important_lore_character_effect = yes }
	#}
```
That's `error.log`
```
Line 699: [00:16:14][jomini_effect.cpp:557]: Unknown effect martial at  file: history/characters/10000_orc.txt line: 342
	Line 700: [00:16:14][jomini_effect.cpp:557]: Unknown effect diplomacy at  file: history/characters/10000_orc.txt line: 342
	Line 701: [00:16:14][jomini_effect.cpp:557]: Unknown effect stewardship at  file: history/characters/10000_orc.txt line: 342
	Line 702: [00:16:14][jomini_effect.cpp:557]: Unknown effect intrigue at  file: history/characters/10000_orc.txt line: 342
	Line 704: [00:16:14][jomini_effect.cpp:557]: Unknown effect learning at  file: history/characters/10000_orc.txt line: 342
	Line 707: [00:16:14][jomini_effect.cpp:557]: Unknown effect 511.8.11 at  file: history/characters/10000_orc.txt line: 345
	Line 709: [00:16:14][jomini_effect.cpp:557]: Unknown effect 511.8.11 at  file: history/characters/10000_orc.txt line: 349
	Line 711: [00:16:14][jomini_effect.cpp:557]: Unknown effect 531.8.11 at  file: history/characters/10000_orc.txt line: 357
	Line 713: [00:16:14][jomini_effect.cpp:557]: Unknown effect 591.11.1 at  file: history/characters/10000_orc.txt line: 358
	Line 715: [00:16:14][jomini_effect.cpp:557]: Unknown effect 612.12.2 at  file: history/characters/10000_orc.txt line: 369
	Line 717: [00:16:14][jomini_effect.cpp:557]: Unknown effect dna at  file: history/characters/10000_orc.txt line: 375
	Line 718: [00:16:14][jomini_effect.cpp:557]: Unknown effect martial at  file: history/characters/10000_orc.txt line: 378
	Line 720: [00:16:14][jomini_effect.cpp:557]: Unknown effect diplomacy at  file: history/characters/10000_orc.txt line: 378
	Line 721: [00:16:14][jomini_effect.cpp:557]: Unknown effect stewardship at  file: history/characters/10000_orc.txt line: 378
```